src/ts3init_target.c

```diff
@@ -47,7 +47,7 @@ ts3init_send_ipv6_reply(struct sk_buff *oldskb, const struct xt_action_param *pa
     struct udphdr *udp;
     struct flowi6 fl;
     struct dst_entry *dst = NULL;
-    struct net *net = dev_net((par->in != NULL) ? par->in : par->out);
+    struct net *net = dev_net((xt_in(par) != NULL) ? xt_in(par) : xt_out(par));
```

Comment:
syntax changed with version 4.10.0, please add a version ifdef like in compat_xtables.h

Reply:
done